### ISSUE TITLE
Consolidate and migrate issues

### ISSUE THREAD

**Issue by hellais** [ooni/ooni.org#317]
As discussed during the last call we are going to be consolidating our issues into single thematic repositories.

This is a problem because:
* It adds cognitive load on us when we have to decide a) where to file an issue b) where to search for an issue we have previously filed
* It makes the performance of the zenhub boards very bad and make the loading time of the workspace bad
* It make it not so nice when we do our planning and retrospective that we have to filter through many different repos in order to plan for each theme (backend, measurements, apps)

In particular we are going to merge:
* ooni/pipeline
* ooni/api
* ooni/sysadmin
* ooni/backend
* ooni/orchestra

Into a single issue repository for all backend related issues.

We are then going to merge all measurements related issues:
* ooni/probe-engine
* ooni/netx
* ooni/jafar
* ooni/spec

For the time being each user-facing product will have it's own issue tracker, but all probe related issue trackers will also get merged.

The differentiation will be solely based on github labels.
 
This means that for the frontend we will eventually end up with:
* ooni/probe (putting ooni/probe-cli and ooni/probe-desktop in here)
* ooni/explorer
* ooni/run
* ooni/design-system

We probably want to keep the issue tracker still enabled on all repos and if users file issues on the wrong repo, we just move them to the correct one.

To counter the fact that this will mean issues do not appear on the zenhub board, we are going to recommend that you always link a PR to the relevant issue if you want it to appear on the board, but this is not a strict requirement.

Unfortunately the native github feature doesn't support API or batch issue transfer, so I am going to be looking into doing this via some form of scripting.

cc @sarathms @bassosimone @FedericoCeratto @agrabeli @lorenzoPrimi

**Comment by hellais** [comment]
Did some research on existing tooling to automate the transfer.

There are many tools which do it by just duplicating the issue, but not many which rely on the native issue transfer (which is I think what we want as it sets up a 302 redirect from the old issue to the new).

The following two look quite promising:
* https://github.com/imphil/github-issue-transfer
* https://github.com/mkArtakMSFT/GitHubIssueTransferExtension

**Comment by hellais** [comment]
I have migrated all issues from:

ooni/pipeline
ooni/api
ooni/sysadmin

into: ooni/backend and applied the appropriate labels.

For ooni/sysadmin I have left all the `incident` issues in the original repo to minimize clutter. For ooni/pipeline I have left issues with the `fingerprintdb` label in place for the same reason.

Moreover, I have put ooni/probe-cli and ooni/probe-desktop in ooni/probe.

For each of these the appropriate component labels have been set and for the ones inside of the current cycle I have also set the zenhub effort estimates and the milestone for the sprint.

As far as I can tell most things were migrated properly by the tool https://github.com/imphil/github-issue-transfer (including labels, assignee and project boards), but all zenhub related metadata has been lost.

This means any sort of pipeline ordering and effort estimates done in zehub will have to be re-applied to the issues.

I don't think this is a big deal as we didn't really do much of that, but it's a useful thing to keep in mind.
